counter.py

```diff
@@ -19,13 +19,20 @@ def minus(self):
     def update_label(self):
         count_label.config(text=str(self.count))
 
+
+
 counter = Counter()
 
 root = tk.Tk()
 root.title("PAScount")
 root.geometry("400x400")
 root.configure(background="#F5F5F5")
 
+# add an image background
+background_image = tk.PhotoImage(file="dog.png")
```

Comment:
dog.png needs changing for image name located in the same directory or the full path to image